src/style/Components.css

```diff
@@ -1,3 +1,7 @@
+html,body {
```

Comment:
This scares me that you're adding this to the entire html body. Is it necessary there?

Reply:
I think it's a very common practice. Should not harm.